### ISSUE TITLE
Request ResNet50 Code Release

### ISSUE THREAD

**Issue by YuchuanTian** [boone891214/GaP#1]
I am very interested in your work. May I have your ResNet50 pruning code? Thank you!

**Comment by LopezCastroRoberto** [comment]
Yes, I am also interested in your work, and I would really appreciate it if I may have your ResNet-50 pruning code.
